dotCMS/src/main/java/com/dotcms/system/event/local/type/staticpublish/StaticPublishEndEvent.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.system.event.local.type.staticpublish;
+
+import com.dotcms.publisher.business.PublishQueueElement;
+import com.dotcms.system.event.local.type.publish.PublishEvent;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Object used to represent an event to be triggered when the static publishing process finishes
+ *
+ * @author Oscar Arrieta
+ */
+public class StaticPublishEndEvent extends PublishEvent {
+
+    public StaticPublishEndEvent(List<PublishQueueElement> publishQueueElements) {
+        this.setName(StaticPublishEndEvent.class.getCanonicalName());
+        this.setPublishQueueElements(publishQueueElements);
+        this.setDate(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)